docs/sphinx/using/backends/backends.rst

```diff
@@ -0,0 +1,29 @@
+CUDA Quantum Backends
+**********************
+
+.. toctree::
+   :caption: Backend Targets
+   :maxdepth: 1
+
+      Simulation <simulators.rst>
+      Hardware <hardware.rst>
+      Multi-Processor Platforms <platform.rst>
+
+**The following is a comprehensive list of the available targets in CUDA Quantum:**
+
+* :ref:`density-matrix-cpu <default-simulator>`
+* :ref:`ionq <ionq-backend>`
+* :ref:`iqm <iqm-backend>`
+* :code:`nvcf`
+* :ref:`nvidia <nvidia-backend>`
+* :ref:`nvidia-fp64 <nvidia-fp64-backend>`
+* :ref:`nvidia-mqpu <nvidia-mgpu-backend>`
+* :ref:`nvidia-mqpu-fp64 <nvidia-mgpu-backend>`
+* :ref:`oqc <oqc-backend>`
+* :code:`orca`
```

Comment:
```suggestion
```
@anthony-santana Please create a github issue to track the missing docs for orca and photonics

Reply:
Issue #1388 and #1389 have been filed and I've removed them from this list